File: ServicesAndRecycler/app/src/main/java/com/example/gonzalomonzon/servicesandrecycler/ui/MainActivity.java
package com.example.gonzalomonzon.servicesandrecycler.ui;

import android.content.Intent;
import android.support.v7.app.AppCompatActivity;
import android.os.Bundle;
import android.view.View;
import android.widget.Button;

import com.example.gonzalomonzon.servicesandrecycler.R;

public class MainActivity extends AppCompatActivity {

    private Button btnOk;
    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
        btnOk=findViewById(R.id.main_btnOk);
        btnOk.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                Intent intent=new Intent(getApplicationContext(),ListPeople.class);
                startActivity(intent);
            }
        });
    }
}

File: MyApplicationWithKotlin/app/src/main/java/com/example/gonzalomonzon/myapplicationwithkotlin/MainActivity.kt
package com.example.gonzalomonzon.myapplicationwithkotlin

import android.os.Bundle
import android.support.v7.app.AppCompatActivity
import org.jetbrains.anko.longToast
import org.jetbrains.anko.toast

class MainActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        varDefinitionString()


    }
    fun varDefinitionString(){

        var vari:String?=null //definicion de una  variable sin asignar

        var vari1="asdasdds" //definicion y asignacion de una variable

        toast("$vari asdasadsaasdads $vari1")

        toast(vari+" "+vari1)

        vari="asdaddasd"
        longToast(vari)

        val url="hjs.com"
        toast(url)
    }
}
